plugin/index.php

```diff
@@ -19,13 +23,26 @@ function gutenberg_menu() {
 	);
 }
 
+// Hook.
+add_action( 'admin_enqueue_scripts', 'gutenberg_scripts_and_styles' );
```

Comment:
Here's a couple examples of the precedent I was referring to:

- https://github.com/WordPress/WordPress/blob/9f4bbcd/wp-content/themes/twentyseventeen/functions.php#L530-L542
- https://github.com/WordPress/WordPress/blob/dd6da70/wp-content/plugins/hello.php#L80

There's not a whole lot of examples in the core code itself, as callbacks are often defined separate from the hook itself or the hook is created in the constructor of a class.